mycpp/gc_str.cc

```diff
@@ -462,3 +468,105 @@ List<Str*>* Str::split(Str* sep) {
 
   return result;
 }
+
+static inline Str* _StrFormat(const char* fmt, va_list args) {
+  int n = strlen(fmt);
```

Comment:
this strlen() could also be moved to the 2 callers -- in one case it passes len() which is pre-computed, in the other it calls strlen()